public/js/us-map.js

```diff
@@ -48,6 +41,9 @@ const plotCities = (response) => {
       .enter()
       .append("circle")
       .attr("cx", d => {
+        count++
```

Comment:
@brittanystoroz it seems like this will only run 150 times, but I need it to run way more- why would that happen?  how could I get around it?